### PR TITLE
CY-3253 Stabilise cluster status test

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cosmo_tester/test_suites/cluster/full_cluster_test.py [hunk 1]
```diff
@@ -398,35 +407,46 @@ def _verify_status_when_rabbit_inactive(broker1, broker2, broker3, logger,
...
-    logger.info('Stopping the other rabbit nodes')
-    broker2.run_command('systemctl stop cloudify-rabbitmq', use_sudo=True)
-    broker3.run_command('systemctl stop cloudify-rabbitmq', use_sudo=True)
-    time.sleep(10)
 
+# It can take time for prometheus state to update.
+# Thirty seconds should be much more than enough.
+@retrying.retry(stop_max_attempt_number=15, wait_fixed=2000)
+def _validate_status_when_all_rabbits_inactive(logger, client):
+    logger.info('Checking status reporter with all syncthing down...')
```
Comment (mateumann) [inline cosmo_tester/test_suites/cluster/full_cluster_test.py:439]: I guess it's not about syncthing, or is it?

Comment (mateumann) [reply cosmo_tester/test_suites/cluster/full_cluster_test.py:439]:
```suggestion
    logger.info('Checking status reporter with all rabbits down...')
```

Comment (geokala) [reply cosmo_tester/test_suites/cluster/full_cluster_test.py:439]: /Everything/ is about syncthing.

Except this, and some other things. Good catch.

Comment (geokala) [reply cosmo_tester/test_suites/cluster/full_cluster_test.py:439]: Updated.

**Review by mateumann — APPROVED** [review]

**Pull request merged**